using_images/db_images/mariadb.adoc

```diff
@@ -0,0 +1,754 @@
+[[using-images-db-images-mariadb]]
+= MariaDB
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[mariadb-overview]]
+== Overview
+{product-title} provides a container image for running MariaDB. This image can
+provide database services based on username, password, and database name
+settings provided via configuration.
```

Comment:
Dopey suggesting using "via" with caution. s/'settings in a configuration file (??)